src/Common/src/CoreLib/System/IO/FileStream.cs

```diff
@@ -304,15 +304,15 @@ public override int Read(byte[] array, int offset, int count)
                 ReadSpan(new Span<byte>(array, offset, count));
         }
 
-        public override int Read(Span<byte> destination)
+        public override int Read(Span<byte> buffer)
```

Comment:
A bunch of these changes under Common aren't going to be validated by CI.  I suggest you first separately submit a PR to coreclr to fix up the corelib APIs.  You'll also want to do one for the non-shared corelib APIs in corert.

Reply:
There isnt any stream.cs file in corert except for the common folder